contracts/wallet.sol

```diff
@@ -574,29 +568,16 @@ contract Wallet is ENSResolvable, AddressWhitelist, SpendLimit, GasTopUpLimit, L
         _;
     }
 
-    /// @dev This is a bulk transfer convenience function, used to migrate contracts.
-    /// @notice If any of the transfers fail, this will revert.
-    /// @param _to is the recipient's address, can't be the zero (0x0) address: transfer() will revert.
-    /// @param _assets is an array of addresses of ERC20 tokens or 0x0 for ether.
-    function bulkTransfer(address payable _to, address[] calldata _assets) external onlyOwnerOrSelf {
-        // check to make sure that _assets isn't empty
-        require(_assets.length != 0, "asset array is empty");
-        // This loops through all of the transfers to be made
-        for (uint256 i = 0; i < _assets.length; i++) {
-            uint256 amount = _balance(_assets[i]);
-            // use our safe, daily limit protected transfer
-            transfer(_to, _assets[i], amount);
-        }
-
-        emit BulkTransferred(_to, _assets);
-    }
-
     /// @dev This function allows for the controller to relay transactions on the owner's behalf,
     /// the relayed message has to be signed by the owner.
     /// @param _nonce only used for relayed transactions, must match the wallet's relayNonce.
     /// @param _data abi encoded data payload.
     /// @param _signature signed prefix + data.
-    function executeRelayedTransaction(uint256 _nonce, bytes calldata _data, bytes calldata _signature) external onlyController {
+    function executeRelayedTransaction(
```

Comment:
Not sure why the formatter is breaking down long lines like this. I set it to a 160 character limit in .prettierrc instead of 80 character limit. Maybe the new version ignores it.

Reply:
Which one looks better? I am used to long lines of code but I thought that breaking lines is maybe a  preferred approach and that's why it is enforced by the formatter.

Reply:
Well some formatters enforce the 80 character limit - I personally don't use the 80 character limit - so I set it to a 160 limit but it seems that it doesn't apply with the latest version.